modules/apps/portal-security/portal-security-auth-test/src/testIntegration/java/com/liferay/portal/security/auth/test/AuthenticatedSessionManagerUtilTest.java

```diff
@@ -129,6 +134,16 @@ public void testRememberMeLogout() throws Exception {
 			CookiesConstants.CONSENT_TYPE_FUNCTIONAL, cookie,
 			mockHttpServletRequest, mockHttpServletResponse);
 
+		ThemeDisplay themeDisplay = ThemeDisplayFactory.create();
+
+		Group group = GroupTestUtil.addGroup();
+
+		themeDisplay.setCompany(CompanyTestUtil.addCompany());
```

Comment:
@manuelecastro is it necessary to create a whole new Company for this test? It bears a high cost.

Reply:
Moving my comment to the latest PR...